src/cocotb_tools/config.py

```diff
@@ -207,32 +207,44 @@ def _get_parser() -> argparse.ArgumentParser:
         help="Print the version of cocotb",
     )
 
+    parser.add_argument(
+        "--posix-paths",
+        action="store_true",
+        help="Output paths using POSIX notation, overriding system-specific notation",
```

Comment:
```suggestion
        help="Use POSIX notation for paths, overriding system-specific notation",
```
(works around "Output" being read as a noun by e.g. me :))